README.md

```diff
@@ -11,14 +11,7 @@ Then the app in development should be accessible on http://localhost:5000/
 
 Now you can make changes to files in `frontend` and the changes **should** show up in the browser automagically.
 
-To create a local admin user you should run this.
-
-```
-docker exec -it canvas_app_explorer ./manage.py shell -c "from django.contrib.auth.models import User; User.objects.create_superuser('admin', 'admin@example.com', 'admin')"
-```
-
-Please see the Wiki for instructions on configuring with LTI.
-
+This app can currently only be launched via LTI.  Please see the Wiki for instructions on configuring with LTI.
```

Comment:
Since the admin role is solely defined in CAE Django database, we may need to add an instruction line here. 

Can include a sql update statement too: `update auth_user set is_staff=1 where id=x;`

Reply:
I put this in the instructions in the notes how you can update this with the application. That's **possibly** better than requesting the user login and run SQL. Basically is the same action. I figured this would just be in the wiki. 

`UPDATE_USERNAME="jonespm@umich.edu";docker exec -it canvas_app_explorer ./manage.py shell -c "from django.contrib.auth.models import User; User.objects.update_or_create(username='$UPDATE_USERNAME', defaults=dict(is_superuser=True, is_staff=True))"`